src/stratigraphy/util/draw.py

```diff
@@ -107,6 +137,20 @@ def draw_coordinates(page: fitz.Page, coordinates: Coordinate) -> None:
         page.draw_rect(coordinates.rect, color=fitz.utils.getColor("purple"))
 
 
+def draw_groundwater_information(page: fitz.Page, groundwater_information: GroundwaterInformation) -> None:
+    """Draw a bounding box around the area of the page where the coordinates were extracted from.
+
+    Args:
+        page (fitz.Page): The page to draw on.
+        groundwater_information (GroundwaterInformation): The groundwater information to draw.
+    """
+    if isinstance(groundwater_information, dict):
+        groundwater_information = GroundwaterInformation(**groundwater_information)
+
+    if (page.number + 1) == groundwater_information.page:  ## page.number is 0-based
+        page.draw_rect(groundwater_information.rect, color=fitz.utils.getColor("pink"))
```

Comment:
Will need to be adjusted after https://github.com/swisstopo/swissgeol-boreholes-dataextraction/pull/73 is merged

Reply:
Yes, indeed.

Reply:
Merged the code.